packages/OS400/curl.inc.in

```diff
@@ -1821,6 +1821,8 @@
      d                 c                   00007
      d  CURLMOPT_MAX_PIPELINE_LENGTH...
      d                 c                   00008
+     d  CURLMOPT_MAX_CONCURRENT_STREAMS...
+     d                 c                   00009
```

Comment:
this is wrong it has to be appended 
     d  CURLMOPT_MAX_CONCURRENT_STREAMS...
     d                 c                   00016

Reply:
```
     d  CURLMOPT_PUSHDATA...
     d                 c                   10015
     d  CURLMOPT_MAX_CONCURRENT_STREAMS...
     d                 c                   10016
      *
      * Bitmask bits for CURLMOPT_PIPELING.
      *

```
Is this correct ?